### PR TITLE
docs: Adaptive Scoring System — auto-detect proven winners by asset class, regime & symbol

### PR DESCRIPTION
**Pull request by eltonaguiar**
## Summary

Comprehensive design document for replacing static scoring weights with **per-cohort Bayesian adaptive weights** that continuously learn which score components predict winners, segmented by asset class, market regime, and symbol tier.

### Problem
Current PF is **0.97** (losing money) with static weights. Score >60 band has 61% WR and 213% PnL, but the system takes too many low-quality picks. Weights don't adapt when market conditions change.

### Solution: Bayesian Adaptive Weight Optimizer (BAWO)

**Core concept**: Each (asset_class, regime, symbol_tier) cohort maintains its own weight distribution over the 10 score components (forward_wr, ml_score, confidence, regime_match, etc.). Weights are updated via Thompson Sampling with Bayesian Linear Regression as trades resolve.

**Key features**:
- **Per-cohort learning**: crypto-bear-altcoin learns different weights than forex-trending-major
- **Exponential time decay** (30-day half-life): recent performance weighted more heavily
- **Per-symbol fine-tuning**: high-frequency symbols (>50 trades) get additive weight corrections
- **Hierarchical fallback**: symbol → cohort → parent → global (handles cold start)
- **Overfitting safeguards**: min 30 trades, regularization toward uniform, 5% max shift rate, weekly holdout validation

**Estimated impact**: PF 1.2-1.5 with adaptive weights alone; PF 1.5-2.0 combined with quick-win fixes.

### Quick Wins (immediate PF improvement)
1. Fix `validation_metrics.js` transaction cost bug (`{ cost }` → `{ total: cost }`)
2. Fix permutation zero-PnL win/loss inconsistency
3. Add score >50 global floor (54% WR vs 43% unfiltered)
4. Disable C-Tier crypto (PF 0.77, -113% PnL)
5. Asset-class PF circuit breaker (auto-disable PF <0.5)

### Implementation phases
1. **Offline validation** — extract component scores from 3430 closed trades, backtest
2. **Shadow mode** — compute adaptive scores alongside current, monitor
3. **Gradual rollout** — blend 70% static + 30% adaptive
4. **Full adaptive** — switch when validation confirms improvement

See `docs/ADAPTIVE_SCORING_SYSTEM.md` for the full design with code samples, architecture diagram, and file inventory.

<div><a href="https://cursor.com/agents/bc-983a3f8d-5383-4a61-b1a2-34cad07650b7"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/assets/images/open-in-web-dark.png"><source media="(prefers-color-scheme: light)" srcset="https://cursor.com/assets/images/open-in-web-light.png">Open in Web</picture></a>&nbsp;<a href="https://cursor.com/background-agent?bcId=bc-983a3f8d-5383-4a61-b1a2-34cad07650b7"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/assets/images/open-in-cursor-dark.png"><source media="(prefers-color-scheme: light)" srcset="https://cursor.com/assets/images/open-in-cursor-light.png">Open in Cursor</picture></a>&nbsp;</div>

### CODE REVIEW

**Comment by eltonaguiar** [comment]
Reviewed. Pure design doc — single new file at `docs/ADAPTIVE_SCORING_SYSTEM.md` (+384 lines, 0 deletions). Describes a Bayesian Adaptive Weight Optimizer (BAWO) for per-cohort adaptive scoring. Substantive technical proposal but doc-only — no code touched, no behavior change. Safe to merge as a design document for future implementation. The actual implementation would need its own PR with the rehab plan §A.4 promotion gate and MERCURYPROMPT.md methodology requirements applied. Merging the design doc.
